aws-rds-cfn-test-common/src/main/java/software/amazon/rds/test/common/schema/ResourceDriftTestHelper.java

```diff
@@ -0,0 +1,139 @@
+package software.amazon.rds.test.common.schema;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Objects;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import org.assertj.core.api.Assertions;
+import org.json.JSONObject;
+
+import com.api.jsonata4java.expressions.EvaluateException;
+import com.api.jsonata4java.expressions.Expressions;
+import com.api.jsonata4java.expressions.ParseException;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableSet;
+
+public class ResourceDriftTestHelper {
+
+    private final static Set<Class<?>> primitiveTypes = ImmutableSet.of(
+            Integer.class,
+            String.class,
+            Boolean.class
+    );
+
+    private static final String PROPERTIES = "/properties";
+    private static final String PROPERTY_OR_SPLIT_REGEX = "\\s*\\$OR\\s*";
```

Comment:
is expression line `hello$ORworld` a valid expression? I am mainly asking due to `*` qualifier instead of `+`

Reply:
Fixed with `+`